packages/table-next/src/react/index.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  SortAscIcon,
+  SortDescIcon,
+  SortIcon
+} from '@pluralsight/ps-design-system-icon'
+import {
+  names as themeNames,
+  useTheme
+} from '@pluralsight/ps-design-system-theme'
+import { HTMLPropsFor, ValueOf } from '@pluralsight/ps-design-system-util'
+import { compose, css } from 'glamor'
+import React from 'react'
+
+import stylesheet from '../css'
+import { aligns, sorts } from '../vars'
+
+const styles = {
+  container: (
+    themeName: ValueOf<typeof themeNames>,
+    opts: { scrollable?: boolean }
+  ) =>
+    compose(
+      css(stylesheet['.psds-table__container']),
+      opts.scrollable &&
+        compose(
+          css(stylesheet['.psds-table__container--scrollable']),
+          css(
+            stylesheet[
+              `.psds-table__container--scrollable.psds-theme--${themeName}`
+            ]
+          )
+        )
+    ),
+  table: (themeName: ValueOf<typeof themeNames>) =>
+    compose(
+      css(stylesheet['.psds-table']),
+      css(stylesheet[`.psds-table.psds-theme--${themeName}`])
+    ),
+  body: () => css(stylesheet['.psds-table__body']),
+  cell: (opts: { align: ValueOf<typeof aligns> }) =>
+    compose(
+      css(stylesheet['.psds-table__cell']),
+      css(stylesheet[`.psds-table__cell--align-${opts.align}`])
+    ),
+  head: () => css(stylesheet['.psds-table__head']),
+  header: (opts: {
+    align: ValueOf<typeof aligns>
+    sortable: boolean
+    sticky: boolean
+  }) =>
+    compose(
+      css(stylesheet['.psds-table__header']),
+      css(stylesheet[`.psds-table__header--align-${opts.align}`]),
+      opts.sortable && css(stylesheet['.psds-table__header--sortable']),
+      opts.sticky && css(stylesheet['.psds-table__header--sticky'])
+    ),
+  sortIcon: () => css(stylesheet['.psds-table__header__sort-icon']),
+  row: (opts: { expanded: boolean; selected: boolean }) =>
+    compose(
+      css(stylesheet['.psds-table__row']),
+      !opts.expanded && css(stylesheet['.psds-table__row--collapsed']),
+      opts.selected && css(stylesheet['.psds-table__row--selected'])
+    )
+}
+
+interface TableStatics {
+  Body: typeof TableBody
+  Cell: typeof TableCell
+  Head: typeof TableHead
+  Header: typeof TableHeader
+  Row: typeof TableRow
+}
+
+interface TableProps extends HTMLPropsFor<'table'> {
+  renderContainer?: React.FC<React.HTMLAttributes<HTMLElement>>
+  scrollable?: boolean
+}
+const Table: React.FC<TableProps> & TableStatics = props => {
+  const {
+    renderContainer: Container = defaultRenderContainer,
```

Comment:
What are you anticipating by the ability here to change from a div container?

Reply:
mostly exposing the div so consumers can add props(styles, aria, testids, etc) and refs when needed